install/wavefront/README.md

```diff
@@ -9,23 +9,21 @@ for installing the Wavefront by VMware adapter on your Kubernetes deployment.
 
 To deploy this adapter, you will need a cluster with the following minimum setup.
 
-* Kubernetes v1.10.0
-* Istio v1.0.3
-* Helm v2.10.0
+* Kubernetes v1.15.0
+* Istio v1.4 or v1.5
```

Comment:
nit:  this can just say v1.4 (since description refers to this as the minimum version supported).

Reply:
I guess i did this based on previous review comment.